docs/topics/pagination.txt

```diff
@@ -161,6 +161,17 @@ Optional arguments
     Whether or not the first page is allowed to be empty.  If ``False`` and
     ``object_list`` is  empty, then an ``EmptyPage`` error will be raised.
 
+``request``
+    An ``HttpRequest`` object, used for creating pagination query strings.
+
+    See :meth:`~Page.next_page_querystring`.
+
+``page_kwarg``
```

Comment:
I find this name a bit confusing. It's a bit verbose, but something like `page_querystring_key_name` might be better.